games-fps/urbanterror/urbanterror-4.3.4_p20211208.ebuild

```diff
@@ -1,34 +1,37 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI="7"
+EAPI="8"
+
+COMMIT_ID="c988f7669bd70b59beb384563d4c8304062ae011"
 
 inherit desktop flag-o-matic toolchain-funcs xdg-utils
 
 DESCRIPTION="Hollywood tactical shooter based on the ioquake3 engine"
-HOMEPAGE="https://urbanterror.info https://github.com/mickael9/ioq3"
+HOMEPAGE="https://www.urbanterror.info/ https://github.com/mickael9/ioq3"
 
 if [[ ${PV} == *9999 ]]; then
 	inherit git-r3
 	EGIT_REPO_URI="https://github.com/mickael9/ioq3.git"
 	EGIT_BRANCH="urt"
 else
-	COMMIT_ID="0429c03056720523d27ca71d5a4aa3e8d00709e7"
 	SRC_URI="https://github.com/mickael9/ioq3/archive/${COMMIT_ID}.tar.gz -> ${P}.tar.gz"
 	S="${WORKDIR}/ioq3-${COMMIT_ID}"
-	KEYWORDS="~amd64 ~x86"
 fi
 
+KEYWORDS="~amd64 ~x86"
 LICENSE="GPL-2"
 SLOT="0"
 IUSE="+altgamma +client +curl debug mumble openal +opus server +skeetshootmod voip vorbis"
-REQUIRED_USE="|| ( client server )
-		voip? ( opus )"
+REQUIRED_USE="
+	|| ( client server )
```

Comment:
Even though that is a non-semantic change, I like this style better. So that change is fine by me.